contrib/vcloud-benchmark.py

```diff
@@ -46,12 +48,21 @@ def download_required_jars(config):
         cmd += ["-warn"]
     cmd += ["-retrieve", "lib/vcloud-jars/[artifact](-[classifier]).[ext]"]
 
-    # install vcloud jar and dependencies
-    subprocess.run(
-        cmd,
-        cwd=_ROOT_DIR,
-        shell=vcloudutil.is_windows(),  # noqa: S602
-    )
+    # Provide temporary directory
+    if config.freshIvyCache:
+        temp_dir = tempfile.mkdtemp()
+        cmd += ["-cache", str(temp_dir)]
+        logging.debug(f"Using fresh ivy cache: {temp_dir}")
+    try:
+        # install vcloud jar and dependencies
+        subprocess.run(
+            cmd,
+            cwd=_ROOT_DIR,
+            shell=vcloudutil.is_windows(),  # noqa: S602
+        )
+    finally:
+        if "temp_dir" in locals():
+            shutil.rmtree(temp_dir)
```

Comment:
Better use `tempfile.TemporaryDirectory` and its `cleanup()` function instead of manual implementation.